### DISCUSSION TITLE
Teams chat

### DISCUSSION THREAD

**Discussion by dkaaven** [O365/python-o365#741] [General]
I have tried to look and search to find the answer, so hope I'm not missing something obvious.

I would love to have the feature where you create a teams chat, even the possiblilities from the beta to reply to chat.

**And then I found the answer..**
Please delete post...

**Comment by alejcas** [comment]
This is posible (ms graph allows that) but it's not implemented on O365. PR are very welcome

**Reply by dkaaven** [reply]
I would love to help, I'm still learning how to program with python, but I'll try to look into it.
